frontend/src/metabase/admin/people/components/PeopleList.jsx

```diff
@@ -219,7 +219,13 @@ export default class PeopleList extends Component {
 
         {hasUsers && (
           <div className="flex align-center justify-between p2">
-            <div className="text-medium text-bold">{t`${total} people found`}</div>
+            <div className="text-medium text-bold">
```

Comment:
Maybe use the opportunity to convert this div and its daddy into Styled Components?

Reply:
@gusaiani thanks for the review! 
I agree that it is good to rewrite code according to our new conventions. I think it should be reasonable and a change is relatively small like a bugfix, wording change, or just small addition, refactoring does not really bring the value we aiming for in the first place which is 10x development velocity. What do you think?